private/buf/cmd/buf/command/breaking/breaking.go

```diff
@@ -400,3 +434,30 @@ func getExternalPathsForImages(imageConfigs []bufwire.ImageConfig, excludeImport
 	}
 	return stringutil.MapToSlice(externalPaths), nil
 }
+
+func mergeImageConfigs(
```

Comment:
Logic like this should not be in a specific command's package.

Reply:
Agreed. I'll refactor once we finalise this